_data/simple-icons.json

```
@@ -12070,6 +12070,16 @@
             "hex": "EC008C",
             "source": "https://www.veepee.fr/"
         },
+        {
+            "title": "Vega",
+            "hex": "2450B2",
+            "source": "https://github.com/vega/logos/blob/af32bc29f0c09c8de826aaafb037935fb69e960a/assets/VG_Black.svg",
+            "guidelines": "https://github.com/vega/logos",
+            "license": {
+                "type": "custom",
```

The Vega license _in the `logos` repo_ passes the [SPDX license checker](https://tools.spdx.org/app/check_license/) for me. I think we should use `BSD-3-Clause` here:

```suggestion
                "type": "BSD-3-Clause",
```